File: crates/algosketch-core/tests/fixtures/two_sum.java
class Algorithms {
    int two_sum(int[] items, int target) {
        for (int i = 0; i < items.length; i = i + 1) {
            for (int j = i + 1; j < items.length; j = j + 1) {
                if (items[i] + items[j] == target) {
                    return i;
                }
            }
        }
        return -1;
    }
}

File: crates/algosketch-core/tests/fixtures/two_sum.go
package main

func two_sum(items []int, target int) int {
	for i := 0; i < len(items); i++ {
		for j := i + 1; j < len(items); j++ {
			if items[i]+items[j] == target {
				return i
			}
		}
	}
	return -1
}
